lib/ansible/modules/cloud/amazon/cloudformation.py

```diff
@@ -451,6 +451,7 @@ def create_changeset(module, stack_params, cfn, events_limit):
                 # Lets not hog the cpu/spam the AWS API
                 time.sleep(1)
             result = stack_operation(cfn, stack_params['StackName'], 'CREATE_CHANGESET', events_limit)
+            result['change_set_id'] = cs['Id']
```

Comment:
Please add this to the module return documentation as well.

Reply:
added return documentation as requested